_includes/feed.liquid

```diff
@@ -38,12 +38,11 @@ section: smartContract
     render = (data) => {
```

Comment:
Have you got an example URL to test this @StephenFluin ?

Reply:
@alexroan `http://localhost:4200/docs/binance-smart-chain-addresses/#BSC%20Testnet%20LINK%20/%20USD` would like right to a pair (new feature), or `http://localhost:4200/docs/binance-smart-chain-addresses/#BSC%20Testnet` for BSC Testnet works.